app/api/attendees.py

```diff
@@ -56,8 +60,13 @@ def before_post(self, args, kwargs, data):
                 "Ticket belongs to a different Event"
             )
         # Check if the ticket is already sold out or not.
-        if get_count(db.session.query(TicketHolder.id).
-                     filter_by(ticket_id=int(data['ticket']), deleted_at=None)) >= ticket.quantity:
+        order_expiry_time = get_settings()['order_expiry_time']
+        if get_count(db.session.query(TicketHolder.id)
+                    .filter(TicketHolder.ticket_id == int(data['ticket']),
+                            TicketHolder.state == 'placed', TicketHolder.deleted_at.is_(None),
```

Comment:
placed or completed

Reply:
@iamareebjamal This is done wrong as TicketHolder has a state which represents its address and not an placed state. I have to change this to order and then check for order state.